kapitan/cli.py

```diff
@@ -656,14 +656,7 @@ def secret_update_validate(args, ref_controller):
                         secret_obj.update_key(awskey)
                         ref_controller[token_path] = secret_obj
 
-            elif token_path.startswith("?{vaultkv:"):
```

Comment:
Is this intended?

Reply:
Yes, this is a cleanup and this code doesn't do anything!